native/proto/README.md

```diff
@@ -0,0 +1,23 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Apache DataFusion Comet: Intermediate Representation of Query Plan
+
+This crate contains code generated from protocol buffer definitions of a query plan intermediate 
+format and is intended to be used as part of the Apache DataFusion Comet project.
```

Comment:
```suggestion
This crate contains the protocol buffer definitions of Spark physical query plans 
and is intended to be used as part of the Apache DataFusion Comet project.
```

Reply:
I think we don't include the generated code in this crate but only the protobuf definitions.